services/rusk.service

```diff
@@ -11,8 +11,7 @@ Environment="RUSK_PROFILE_PATH=/opt/dusk/rusk"
 User=dusk
 WorkingDirectory=/opt/dusk
 
-ExecStartPre=!/bin/bash -c '/opt/dusk/bin/rusk recovery-keys >> /var/log/rusk_recovery.log'
```

Comment:
Don't delete this
Keys needs to be checked

Maybe it's a good idea to enable a flag on the recovery for just check instead of creating them if missed

Can you?